03.bowling/bowling.rb

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'debug'
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2) do |s|
+  frames << s
+end
+
+def delete_frames(frames, first_number, end_number)
+  (first_number..end_number).to_a.each_index do |i|
+    frames.delete_at(end_number - i)
+  end
+end
+
+point = 0
+flg = false
+# １０フレーム目で３投投げた時の処理
+if frames[10]
+  if frames[11]
```

Comment:
2投ともストライクであれば

```suggestion
  if frames[9] == 10 && frames[10] == 10
```

とした方がわかりやすいです